Examples/php/value/runme.php

```diff
@@ -3,15 +3,15 @@
 	require "example.php";
 
 
-	$v = new_vector();
-        vector_x_set($v,1.0);
-        vector_y_set($v,2.0);
-        vector_z_set($v,3.0);
-
-	$w = new_vector();
-        vector_x_set($w,10.0);
-        vector_y_set($w,11.0);
-        vector_z_set($w,12.0);
+	$v = new Vector();
+        $v->__set("x",1.0);
+        $v->__set("y",2.0);
+        $v->__set("z",3.0);
+
+	$w = new Vector();
+        $w->__set("x",10.0);
+        $w->__set("y",11.0);
+        $w->__set("z",12.0);
```

Comment:
There isn't a `$v->x_set()` in the current wrappers. So using the magic methods instead.